scss/_detail.scss

```diff
@@ -1,7 +1,42 @@
+.container {
+  &-detail {
+    height: 100vh;
```

Comment:
This is not necessary since we want the height to be `auto`

Reply:
Done 😄